models/OrderDao.js

```diff
@@ -0,0 +1,40 @@
+import prisma from '../prisma';
+
+const findCartItems = async (id) => {
+  return await prisma.$queryRaw(`
+      SELECT
+        carts.id as cart_item_id,
+        users.id as user_id,
+        products.id,
+        products.name,
+        images.image_url,
+        series.name as series,
+        products.series_number,
+        volumes.volume_ml,
+        volumes.price
```

Comment:
```suggestion
      SELECT
        carts.id as cartItemId,
        users.id as userId,
        products.id,
        products.name,
        images.imageUrl,
        series.name as series,
        products.seriesNumber,
        volumes.volume_ml,
        volumes.price
```
aliasing 할 때 부터 카멜케이스로 해주면, 나중에 service 단이나 controller로 return 하고 나서도 카멜케이스를 사용할 수 있고
return 되는 json도 카멜케이스로 나타나서 편리합니다 :) 👍

Reply:
감사합니다! 카멜케이스로 수정했습니다!

https://github.com/wecode-bootcamp-korea/fullstack1-1st-BaseNote-backend/pull/8/commits/20fc4cae9d221076fb5ce5ac306f9fecbcf1b66e

소헌님, 여기서 질문이 하나 있습니다!
지금까지 프론트에서 사용했던 API의 대다수가 json의 키값이 snake_case였던 것 같아요..!
(예시 : [github 유저정보 조회 API](https://api.github.com/users/iamhayoung))

그래서 암묵적으로 JS에서는 camelCase를 사용하는 것이 보편적인 룰이듯이
API json의 키값은 snake_case를 사용하는 것이 보편적인 룰이 생각했었는데
이부분도 그 회사의 컨벤션에 맞추는 것인가요?

아니면 뭔가 제가 잘못이해하고 있는 부분이 있을까요?

Reply:
아하 네 그것도 맞추기 나름입니다 :) 전혀 상관 없습니다-! 현재 프론트/백엔드가 전부 자바스크립트로 되고 있어서 그렇게 추천드렸습니담:)

Reply:
그렇군요 감사합니다!!!! 그렇다면 제안해주신대로 camelCase로 가겠습니다🙈